gamemodes/terrortown/gamemode/shared/sh_entity_ext.lua

```diff
@@ -0,0 +1,179 @@
+--- DOORS MODULE STUFF ---
```

Comment:
Missing `@author`.
Should start with `---` followed by `-- This is a short description ..... `